ibm/service/iampolicy/resource_ibm_iam_policy_assignment.go

```diff
@@ -21,9 +22,8 @@ import (
 )
 
 const (
-	InProgress = "in_progress"
-	complete   = "complete"
-	failed     = "failed"
+	WAITING = "waiting"
+	READY   = "ready"
```

Comment:
Is this change for the terraform provider end user and now that we are retrying/waiting until is resource is ready?

Reply:
we are using 2 states instead of 3. We are achieving same behavior like waiting for the resource creation

Reply:
Sounds good, that's what it looked like I just wanted to confirm my thinking since I can misunderstand things 😅